### ISSUE TITLE
When running ./autogen.sh, Getting error 

### ISSUE THREAD

**Issue by syedkhizarulhaq-cis** [ntop/nDPI#761]
Hi,

When I run the command ./autogen.sh I am getting the error below:

![Screenshot from 2019-07-25 16-17-18](https://user-images.githubusercontent.com/33244888/61870513-bf3b4b00-aef7-11e9-89a2-bf853afb4473.png)

Thanks

**Comment by themadprofessor** [comment]
Judging by the directory you are in and the messages from `./autogen.sh`, you want [ntopng](https://github.com/ntop/ntopng).

Make sure you have pkg-config installed, and autotools is up to date. It works for me with
```
% autoreconf --version
autoreconf (GNU Autoconf) 2.69
```

**Comment by syedkhizarulhaq-cis** [comment]
Hi @themadprofessor,
I just couldn't get it to run from the github repository that I cloned.
Apparently there is some problem with the autoconf and even after going through most of the links on google regarding the **possibly undefined macro** error, I could not find a fix. :(
![Screenshot from 2019-07-26 10-21-53](https://user-images.githubusercontent.com/33244888/61927725-425bb000-af8f-11e9-9123-3df2ec6d11b8.png)
I also have the pkg-config installed : 
![Screenshot from 2019-07-26 10-23-41](https://user-images.githubusercontent.com/33244888/61927778-8058d400-af8f-11e9-9996-1e2584957b73.png)

So after all of this, the solution that I found was to type the below in your terminal if you have this error and it is annoying you : 
**apt-get install ntopng**

![Screenshot from 2019-07-26 10-27-06](https://user-images.githubusercontent.com/33244888/61927879-f3fae100-af8f-11e9-92c7-6fce38002849.png)

Cheers and Thanks!
